docs/7.x/changelog.md

```diff
@@ -33,22 +33,70 @@ These versions are past their End of Support date, and no longer receive securit
+    * Introduce a hidden flag that eliminates workers from upgrade plan
+    * Introduce a hidden flag to allow parallel execution of certain upgrade phases
+    * Cache plan phases instead of re-querying Etcd each time
+    * Parallelize phases that run on multiple nodes, but do not affect the cluster (e.g. configuriation)
+    * Improve AWS event handler integration
+    * Add debug logging (including response times) to our internal API client
+    * Pre-emptively exit if an expand operation won't be accepted
+    * Allow multiple worker shrink/expand operations to run at the same time
+* Split `tele helm build` from `tele build` ([#1317](https://github.com/gravitational/gravity/pull/1317))
+* Bump Helm 2 version to 2.16.12 ([#2307](https://github.com/gravitational/gravity/pull/2307), [#2310](https://github.com/gravitational/gravity/pull/2310), [planet#789](https://github.com/gravitational/planet/pull/789))
+
+#### Removals
+* Removed support for Debian 8 ([#2210](https://github.com/gravitational/gravity/pull/2210))
+* Removed provisioner resources from the reference `telekube` application image ([#1561](https://github.com/gravitational/gravity/pull/1561))
+* Removed the Gravity's terraform provider ([#1622](https://github.com/gravitational/gravity/pull/1622))
+
+#### Bugfixes
+* Fix an issue with hub certificates missing SANs after expand ([#1323](https://github.com/gravitational/gravity/pull/1323)).
+* Removed stack traces from web API calls ([#2357](https://github.com/gravitational/gravity/pull/2357), [teleport#5070](https://github.com/gravitational/teleport/pull/5070), [trace#60](https://github.com/gravitational/trace/pull/60))
+* Fix a bug where tailing an operation plan would truncate at 100 events ([#2426](https://github.com/gravitational/gravity/pull/2426))
+
+#### Internal Changes
+* Increase CoreDNS CPU and memory allocation [#2430](https://github.com/gravitational/gravity/pull/2430)
+* Replaced in-house drain logic with kubernetes drain logic ([#1652](https://github.com/gravitational/gravity/pull/1652))
+* Remove [Serf](https://www.serf.io/) from the ping/latency checker ([#2234](https://github.com/gravitational/gravity/pull/2234), [planet#765](https://github.com/gravitational/planet/pull/765), [satellite#277](https://github.com/gravitational/satellite/pull/277), [monitoring-app#197](https://github.com/gravitational/monitoring-app/pull/197))
+* Replace [Serf](https://www.serf.io/) for managing cluster membership ([#2329](https://github.com/gravitational/gravity/pull/2329), [planet#794](https://github.com/gravitational/planet/pull/794), [satellite#284](https://github.com/gravitational/satellite/pull/284), [satellite#282](https://github.com/gravitational/satellite/pull/282))
+* Switched from [Dep](https://github.com/golang/dep) to [Go Modules](https://golang.org/ref/mod) ([#1454](https://github.com/gravitational/gravity/pull/1454))
```

Comment:
Same as above.
```suggestion
* Switch from [Dep](https://github.com/golang/dep) to [Go Modules](https://golang.org/ref/mod) ([#1454](https://github.com/gravitational/gravity/pull/1454))
```

Reply:
Fixed all of the noted verb moods in 8b6b022